ibm_catalog.json

```diff
@@ -0,0 +1,185 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-enterprise",
+      "label": "Cloud automation for IBM Enterprise",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "solution",
+        "support_ibm"
```

Comment:
you are missing a category tag - we cannot publish to catalog without a category tag. Perhaps go with "security"?

Remove "support_ibm" - this DA is not yet supported by IBM Cloud - it will be published to the community registry first.